tests/test_go_server.py

```diff
@@ -138,8 +138,9 @@ def run_logger(code_dir, code):
     with open(ROOT / "test-go" / "test.go.tmpl", "r") as fp:
         tmpl_code = fp.read()
 
-    tmpl_code = tmpl_code.replace("/* CODE */", code)
+    tmpl_code = tmpl_code.replace("/* CODE */", code).replace("/* IMPORTS */", imports)
 
+    print(tmpl_code)
```

Comment:
Stray print?

Reply:
TY, that was from getting the new tests working